src/support.js

```diff
@@ -81,27 +81,38 @@ const supportCommands = () => {
 
   Cypress.Commands.add('clickToolbarButton', clickToolbarButton)
 
-
   /**
-   * Check for notices and warnings
+   * Check for PHP notices and warnings in the HTML page source.
+   * This command looks for keywords such as 'Deprecated' in bold followed by a colon.
+   * If such a styled keyword is found, the test fails with the PHP problem message.
    *
-   * @memberof cy
+   * Looking for:
+   * - <b>Warning</b>:
+   * - <b>Deprecated</b>:
+   * - <b>Notice</b>:
+   * - <b>Strict standards</b>:
+   *
+   * @memberof Cypress.Commands
    * @method checkForPhpNoticesOrWarnings
-   * @returns Chainable
+   * @returns {Cypress.Chainable}
```

Comment:
This method returns nothing

Reply:
Yes, this is correct. I have the method documentation rewritten:
```
@returns {void} - It does not return any value, but it is chainable with other Cypress commands.
```